src/main/kotlin/no/nav/lydia/sykefraversstatistikk/import/StatistikkConsumer.kt

```diff
@@ -1,24 +1,27 @@
 package no.nav.lydia.sykefraversstatistikk.import
 
+import SykefraversstatistikkImportDto
+import com.google.gson.GsonBuilder
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.Job
 import kotlinx.coroutines.launch
 import no.nav.lydia.Kafka
-import no.nav.lydia.NaisEnvironment
+import no.nav.lydia.sykefraversstatistikk.SykefraversstatistikkRepository
 import org.apache.kafka.clients.consumer.KafkaConsumer
 import org.apache.kafka.common.errors.RetriableException
 import org.apache.kafka.common.serialization.StringDeserializer
 import org.slf4j.Logger
 import org.slf4j.LoggerFactory
 import java.time.Duration
-import java.time.temporal.ChronoUnit
 import kotlin.coroutines.CoroutineContext
 
 object StatistikkConsumer : CoroutineScope {
     private val logger: Logger = LoggerFactory.getLogger(this::class.java)
     lateinit var job: Job
     lateinit var kafka: Kafka
+    // TODO: Er det greit å holde en datasource oppe i en coroutine scope?
```

Comment:
Tror det skal være ganske uproblematisk, men det kan isåfall fikses når det motsatte bevises